test/modern_bpf/test_suites/syscall_enter_suite/epoll_create1_e.cpp

```diff
@@ -0,0 +1,41 @@
+#include "../../event_class/event_class.h"
+
+#if defined(__NR_epoll_create1) && defined(__NR_close)
+TEST(SyscallEnter, epoll_create1E)
+{
+	auto evt_test = get_syscall_event_test(__NR_epoll_create1, ENTER_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL ===========================*/
+
+	int32_t flags = 0;
```

Comment:
Here I would use `EPOLL_CLOEXEC` just to check that all work well!

```suggestion
	int32_t flags = EPOLL_CLOEXEC;
```